src/filter.ts

```diff
@@ -32,11 +37,24 @@ export interface RangeFilter {
    * @maxItems 2
    * @minItems 2
    */
-  range: any;
+  range?: any;
+  lt?: number;
+  lte?: number;
+  gt?: number;
+  gte?: number;
+
+  negate?: boolean;
 }
 
 export function isRangeFilter(filter: any): filter is RangeFilter {
-  return filter && !!filter.field && isArray(filter.range) && filter.range.length === 2;
+  if (filter && !!filter.field) {
+    if (isArray(filter.range) && filter.range.length === 2) {
+      return true;
+    } else if (XOR(filter.gt !==undefined, filter.gte!==undefined) || XOR(filter.lt!==undefined, filter.lte!==undefined) ) {
```

Comment:
Plus, adding this condition will make the logic fail as combination of `gt` and `gte` will make `getFilterExpression` return just the `filter` on [this line](https://github.com/vega/vega-lite/pull/1467/files#diff-74ca0355251fd3e28659aeb89ca5ad34R40).